it/get-additional-arguments.ps1

```diff
@@ -21,7 +21,7 @@ if ($language -eq "csharp") {
     $command = " --output `"./it/$language/client`" --namespace-name `"app.client`""
 }
 elseif ($language -eq "java") {
-    $command = " --output `"./it/$language/src`""
+    $command = " --output `"./it/$language/src/apisdk`""
```

Comment:
This is, likely, another breaking change of the "default behavior".

Reply:
meaning it'll break other integration tests or did you mean something else?

Reply:
When I implemented the IT tests, the CLI was correctly placing the files in the folder matching the package name, now it's not.

I haven't bisected the issue to see what changed, though, was the previous "default package" null?

Reply:
do you mean the --namespace-name parameter? no, it was always defaulted to "ApiSdk"

Reply:
Ok, strange we have not caught this earlier in this case ...